django_mongodb_backend/gis/features.py

```diff
@@ -39,6 +42,11 @@ def django_test_skips(self):
                     # SouthTexasCity fixture objects use SRID 2278 which is ignored
                     # by the patched version of loaddata in the Django fork.
                     "gis_tests.distapp.tests.DistanceTest.test_init",
+                    "gis_tests.distapp.tests.DistanceTest.test_distance_lookups",
+                    "gis_tests.distapp.tests.DistanceTest.test_distance_lookups_with_expression_rhs",
+                    "gis_tests.distapp.tests.DistanceTest.test_distance_annotation_group_by",
+                    "gis_tests.distapp.tests.DistanceFunctionsTests.test_distance_simple",
+                    "gis_tests.distapp.tests.DistanceFunctionsTests.test_distance_order_by",
```

Comment:
These are already skipped by `@skipUnlessDBFeature("has_Distance_function")` (I imagine you may have been  experimenting with trying to support it).